### PR TITLE
Rewrite SfSlidingSection

### PR DESCRIPTION
**Pull request by przspa**
# Related issue

# Scope of work

Imported hammerjs and better mobile behaviour form SfSlidingSection
# Screenshots of visual changes

# Checklist

- [x] I followed [composition rules](https://docs.storefrontui.io/component-rules.html) for my component
- [x] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

**Comment by filrak** [comment]
How about Storybook? Maybe we should just add a story with full-green image and some text with heading below?

**Comment by filrak** [comment]
Also make sure it work in SSR

File: packages/vue/src/examples/pages/product/Product.vue [hunk 1]
```diff
@@ -616,6 +621,7 @@ export default {
         min-width: calc(
           (375 / 490) * (100vh - 177px)
         ); // (oldWidth / oldHeight) * newHeight = newWidth
```
Comment (filrak) [inline packages/vue/src/examples/pages/product/Product.vue]: are you sure it's transformed to plain css, not css vars?

**Comment by przspa** [comment]
About Storybook
_mobile
_desktop

**Comment by przspa** [comment]
> Also we might want to have a SCSS modifier to distribute content in 2 columns on desktop (instead of below each other )
@filrak but on desktop we have 2 columns?

File: packages/vue/src/examples/pages/product/Product.vue (hunk 1 above)
Comment (przspa) [reply packages/vue/src/examples/pages/product/Product.vue]: Haha, thanks I found an error.
Should be:
``` css
min-width: calc((375 / 490) * (100vh - 177px)); // (oldWidth / oldHeight) * newHeight = newWidth
min-width: calc(((var(--vh, 1vh) * 100) - 177px) * (375 / 490));
```

**Comment by przspa** [comment]
> Also make sure it work in SSR

unsupported, because hammerjs unsupported ssr 👎

**Comment by filrak** [comment]
@przspa can we add mobile padding (with spacer to the slot by default) to make it look a bit better by default/ If you think it's a bad idea you can add just to this story

**Comment by przspa** [comment]
> @przspa can we add mobile padding (with spacer to the slot by default) to make it look a bit better by default/ If you think it's a bad idea you can add just to this story

@filrak Done

**Comment by filrak** [comment]
Thanks!

**Pull request merged**